### DISCUSSION TITLE
Aurora Borealis

### DISCUSSION THREAD

**Discussion by TonAmSi** [aaronwmorris/indi-allsky#1255] [Show and tell]
Here you can see the pictures of the northern lights from Hamburg, Germany.  🤗
https://allsky.astrodon.space/indi-allsky/videoviewer

**Comment by aaronwmorris** [comment]
Very Nice!  It put on quite a show here in Alabama.

https://allsky.aarmor.net/indi-allsky/watch_timelapse?id=932

**Reply by TonAmSi** [reply]
Oh, wow! What a night! 
Yours is so great. And you had a clear sky. Simply mega!

**Comment by caphector** [comment]
I actually got some in the SF/Bay Area! https://www.youtube.com/watch?v=Pkks7PWzpSM

**Comment by aaronwmorris** [comment]
Here is @gordtulloch camera in Winnipeg:

https://gordtulloch.com/indi-allsky/images/ccd_e5d3ba4e-801b-468e-8db6-5fc4c67b3ba4/20240510/allsky-timelapse_ccd2_20240510_night.mp4

https://gordtulloch.com/indi-allsky/

**Comment by aaronwmorris** [comment]
And Ken Hall's camera in Perth, Australia with the Aurora Australis hours before we saw it in North America:

https://allsky.nblhunt.net/indi-allsky/images/ccd_6440f61c-0c55-47e9-bb50-d3c7cadb5e8e/20240510/allsky-timelapse_ccd1_20240510_night.mp4

https://allsky.nblhunt.net/indi-allsky/
